lib/reply.js

```diff
@@ -166,7 +166,7 @@ function onSendEnd (reply, payload) {
   }
 
   if (!reply.res.getHeader('Content-Length')) {
-    reply.res.setHeader('Content-Length', Buffer.byteLength(payload))
+    reply.res.setHeader('Content-Length', Buffer.byteLength(payload).toString())
```

Comment:
`'' + Buffer.byteLength(payload)`

Reply:
@allevo why? `.toString` seems clearer. Is there a performance difference?

Reply:
I haven't benchmarked it, but I suspect there is. not sure it's noticeable.

Reply:
`'' + Buffer.byteLength(payload)` is faster.